hack/e2e-scripts/qa-container-scale.sh

```diff
@@ -51,7 +51,7 @@ launch_containers() {
 
 # Function to print help message
 print_help() {
-  echo "Usage: ./script.sh -kc-password <password> -nexd-password <password> --nexd-count <count>"
+  echo "Usage: ./script.sh --kc-password <password> --nexd-password <password> --nexd-count <count>"
```

Comment:
want to fix `script.sh` while you're at it? You could just replace it with `$0`

Reply:
done